app/controllers/items_controller.rb

```diff
@@ -1,7 +1,7 @@
 class ItemsController < ApplicationController
-  def menu
-    @items = Item.all
-    @categories = Category.all
+  def index
+    @items = item.all
```

Comment:
I don't think this matters too much now since we just have a mock up but is this just a typo to have item.all and category.all? instead of Item and Category?